modules/configuration/src/testFixtures/java/org/apache/ignite/internal/configuration/storage/TestConfigurationStorage.java

```diff
@@ -63,7 +63,7 @@ public TestConfigurationStorage(ConfigurationType type, Map<String, Serializable
 
     @Override
     public void close() {
-        // No-op.
+        listeners.clear();
```

Comment:
Is that OK? Please add a comment, like "we reuse this instance after we call `close`" or something